.github/actions/testmo/testmo-threads-submit-playwright/action.yml

```diff
@@ -0,0 +1,58 @@
+name: submit-run-results-to-testmo
+description: "Sends tests results from run to testmo"
+inputs:
+  testmoUrl:
+    description: "Testmo project URL"
+    required: true
+  testmoToken:
+    description: "Testmo token"
+    required: true
+  testmoRunId:
+    description: "Parallelized job Testmo run id"
+    required: true
+runs:
+  using: composite
+  steps:
+    - uses: actions/setup-node@v3
+      with:
+        node-version-file: ".nvmrc"
+        cache: npm
+    - name: Cache node modules
+      uses: actions/cache@v3
+      env:
+        cache-name: cache-node-modules
+      with:
+        path: ~/.npm
+        key: ${{ runner.os }}-qa-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-qa-${{ env.cache-name }}-
+          ${{ runner.os }}-qa-
+          ${{ runner.os }}-
+
+    - name: Install Dependencies
+      if: steps.cache-node-modules.outputs.cache-hit != 'true'
+      shell: bash
+      run: NODE_OPTIONS=--max_old_space_size=4096 npm install
+
+    - name: Install dependencies
+      if: ${{ ! cancelled() }}
+      working-directory: .github/workflows
+      shell: bash
+      run: npm ci
+
+    - name: Submit results
+      if: always() && ${{ inputs.browsers }} == 'all'
```

Comment:
Isn't the `npm ci` needed by "Submit results"? `npm ci` has the condition `! cancelled()` but "Submit results" has `always()` thus it may crash if the run is cancelled

Reply:
I will add cancelled()